mirdata/annotations.py

```diff
@@ -79,6 +79,26 @@ def __repr__(self):
         return repr_str
 
 
+class MultiAnnotator(Annotation):
+    """Multiple annotator class.
+    This class should be used for datasets with multiple annotators (e.g. multiple annotators per track).
+
+    Attributes:
+        annotators (list): list with annotator ids
+        annotations (list): list of annotations (e.g. [annotations.BeatData, annotations.ChordData]
+    """
+
+    def __init__(self, annotators, annotations) -> None:
```

Comment:
I think these two inputs would be better as a single dictionary:
{ 'annotator-id': Annotation }

same with how it's stored, self.annotations = `{'annotator1': ... , 'annotator2', ...}`
because it will make it easier to look up a specific dictionary instead of having to look for matching indexes.

Reply:
I tried this out and noticed that some tracks have the same `annotator id` for different annotations in the same track (which might be a bit weird actually... but it's there :/). Also the sintax gets a bit messy, since I have to iterate in the keys each time I want to access an annotation  because the annotator id changes from track to track. So I changed back to what was before for now. Let me know what you think